.travis.yml

```diff
@@ -21,6 +21,9 @@ script:
       -w "/data" \
       ${OS_TYPE}:${OS_VERSION} \
       ./tests.sh
+  - shellcheck ood-portal-generator/sbin/update_ood_portal
+  - shellcheck -x nginx_stage/sbin/nginx_stage
```

Comment:
what does -x do

Reply:
for sourcing external (-x) files.  That file runs `source` on /etc/profile and `NGINX_PROFILE`